MTM-53125 Documentation for filter feature on apps, packages and microservices lists. (#1250)
SoftwareAG/c8y-docs: content/users-guide/administration-bundle/managing-applications.md

<img src="/images/users-guide/Administration/admin-all-applications.png" alt="All applications" style="max-width: 100%">

> In the **All applications** tab, you can see all applications available in your tenant. There are two kinds of applications' availability:

Reviewer: ```suggestion
In the **All applications** tab, you can see all applications available in your tenant. There are two types of availability for applications:
```

> - [Subscribed applications](#subscribed-applications) - applications subscribed to the tenant, either provided by the platform (as default applications) or a service provider.

Reviewer: ```suggestion
- [Subscribed](#subscribed-applications) - applications subscribed to the tenant, either provided by the platform (as default applications) or a service provider.
```

> - [Custom applications](#own-applications) - applications owned by the tenant. You can [add custom applications](#adding-applications) in various ways as own applications.

Reviewer: ```suggestion
- [Custom](#own-applications) - applications owned by the tenant. You can [add custom applications](#adding-applications) in various ways as own applications.
```

Packages are combinations of plugins and blueprints which can be packed together into a single file and then be deployed to the platform. Thus, packages offer better shareability and reusability of UI features across different applications and allow to add UI features to applications without coding knowledge.

> Packages can contain two types of contents:

Reviewer: ```suggestion
Packages can contain two types of content:
```
